README.md

```diff
@@ -11,6 +11,7 @@ Calculator ships regularly with new features and bug fixes. You can get the late
 ## Features
 - Standard Calculator functionality which offers basic operations and evaluates commands immediately as they are entered.
 - Scientific Calculator functionality which offers expanded operations and evaluates commands using order of operations.
+- Graphing Calculator functionality which offers fast and precise graphs to almost every mathematical equations in 2-D.
```

Comment:
I'd prefer that we say something shorter and simpler: `- Graphing Calculator for visualizing math equations.`

Reply:
I think mine is better and provide more info about `graphing calculator`